api-client/src/endpoints/submitHelpForm.ts

```diff
@@ -0,0 +1,57 @@
+import NotFoundError from '../errors/NotFoundError'
+import OfferModel from '../models/OfferModel'
+import { MALTE_HELP_FORM_OFFER_ROUTE, SPRUNGBRETT_OFFER_ROUTE } from '../routes'
+import createOffersEndpoint from './createOffersEndpoint'
+
+type SubmitHelpFormParams = {
+  cityCode: string
+  languageCode: string
+  baseUrl: string | (() => Promise<string>)
+}
+
+type SubmitHelpFormReturn = Promise<{
+  helpButtonOffer: OfferModel
+  submit: () => Promise<void>
+}>
+
+const submitHelpForm = async ({ cityCode, languageCode, baseUrl }: SubmitHelpFormParams): SubmitHelpFormReturn => {
```

Comment:
Why did you choose to custom implement this instead of using the endpoint classes (like in createFeedbackEndpoint)?

Reply:
I tried following the pattern of the sprungbrett offers endpoint. I'll try finishing the page based offer pr today, so that I can correctly implement that. Using the offers endpoint should also be perfectly okay, you're right.